templates/music/album_detail.html

```diff
@@ -57,28 +57,6 @@
 
     <script>
         {% if album.track_set.all %}
-            var ctx = document.getElementById('myChart').getContext('2d');
-            var myChart = new Chart(ctx, {
-                type: 'line',
-                data: {
-                    labels: [
-                        {% for month_revenue in estimated_cumulative_revenue %}
```

Comment:
Are we not going to use `estimated_cumulative_revenue` anymore? If so I should remove this from the backend too.